.ci-ignore

```diff
@@ -0,0 +1,2 @@
+python_action
+mkdocs.yml
```

Comment:
If code inspector is to be disabled completely, then, yes, this _.ci-ignore_ file is useless. However, it might be useful for other syntax (besides python). That is completely up to you.

Reply:
Okay, it's fine.